src/packages/steps/steps.tsx

```
@@ -39,7 +39,8 @@ export const Steps: FunctionComponent<
     classPrefix,
     {
       [`${classPrefix}-${direction}`]: true,
-      [`${classPrefix}-${direction}-count-${React.Children.count(children)}`]: true,
+      [`${classPrefix}-${direction}-count-${React.Children.count(children)}`]:
+        true,
```

ESLint

[error] Delete `⏎·······`

(prettier/prettier)